sdk/src/Core/Amazon.Runtime/Internal/Auth/AWS4Signer.cs

```diff
@@ -1229,6 +1229,7 @@ public static AWS4SigningResult SignRequest(IRequest request,
             }
 
             var signedAt = CorrectClockSkew.GetCorrectedUtcNowForEndpoint(request.Endpoint.ToString());
+            request.SignedAt = signedAt;
```

Comment:
Does our Sig V4a signer not handle clock skew today?

Reply:
It does handle clockskew today
https://github.com/aws/aws-sdk-net/blob/main/extensions/src/AWSSDK.Extensions.CrtIntegration/CrtAWS4aSigner.cs#L104
The above code eventually calls `CorrectClockSkew.GetCorrectedUtcNowForEndpoint`  to get the time it was signed at. Would you like me to populate `request.SignedAt` in the crt extensions package as well?

Reply:
Why would we not set `request.SignedAt` for `CrtAWS4aSigner`? Without it wouldn't SigV4a requests rely on the global clockskew value versus the actual SDK request time which is what we are trying to fix with this PR and the previous PR.

Reply:
Just added it to crt as well